mails/themes/modern/core/account.html.twig

```diff
@@ -316,8 +316,6 @@
         </style>
         <![endif]-->
   <!--[if !mso]><!-->
-  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,500,700" rel="stylesheet" type="text/css">
```

Comment:
Hi @lmeyer1 

nice catch on this bug. Just a question did you remove them manually or did you fix the generation in the theme generator?

Any content on these files must be strictly exported via the generator and never modified manually, or else next time we change a wording in theme and re-export it all of these fixes will be removed again.

Reply:
@jolelievre Yes, see https://github.com/PrestaShop/mjml-theme-converter/issues/18.

Reply:
There were already similar imports in the template, are they related?

https://github.com/PrestaShop/PrestaShop/blob/5bf1649548c25ae8c226b0d09d6d3a50b6fed214/mails/themes/modern/core/account.html.twig#L527-L528

Reply:
Yes they are. The import is done in two ways, `<link>` is a fallback for mail clients that don't know `@import`. I have no clue what font import is for which client.